NEWS.md

```diff
@@ -109,6 +109,8 @@ unit = "s")
 
 13. A relatively rare case of segfault when combining non-equi joins with `by=.EACHI` is now fixed, closes [#4388](https://github.com/Rdatatable/data.table/issues/4388).
 
+14. When using `:` operator on a _call_ object(s) inside `j` argument, it could have been improperly recognised as an attempt to select columns (`with=FALSE`) rather than evaluate expression within a data.table, [#2069](https://github.com/Rdatatable/data.table/issues/2069). Thanks to @franknarf1 for reporting.
```

Comment:
Please rephrase the NEWS item, I doubt it will be well understood. I would lead with a concrete example and then explain the behavior.